sdk/python/ydk/providers/_provider_plugin.py

```diff
@@ -287,17 +379,17 @@ def _encode_read_request(self, root, entity, optype, only_config):
         XmlEncoder().encode_filter(entity, root, optype)
         return root
 
-    def _encode_delete_request(self, root, entity):
-        meta_info = entity._meta_info()
-        if self._entity_is_abstract(entity, meta_info):
-            self._raise_parent_hierarchy_error()
-            return
-        NSMAP = { None: meta_info.namespace}
-        root = etree.SubElement(root, meta_info.yang_name, nsmap=NSMAP)
-        root.set('{urn:ietf:params:xml:ns:netconf:base:1.0}operation', 'delete')
-        # are there keys
-        root = self._encode_keys(root, entity, meta_info)
-        return root
+    # def _encode_delete_request(self, root, entity):
+    #     meta_info = entity._meta_info()
```

Comment:
Does this code need to commented out? Can it be deleted if it is not being used?